src/ontology/imports/ncbitaxon_terms.txt

```diff
@@ -1427,3 +1438,34 @@ http://purl.obolibrary.org/obo/NCBITaxon_9922
 http://purl.obolibrary.org/obo/NCBITaxon_9925
 http://purl.obolibrary.org/obo/NCBITaxon_9963
 http://purl.obolibrary.org/obo/NCBITaxon_9989
+http_//purl.obolibrary.org/obo/NCBITaxon_119825
```

Comment:
Typo in URL! `_`!

Reply:
!!! Thank you for catching this! 
I have fixed the typos